images/airflow/2.9.2/docker-compose-hybrid.yaml

```diff
@@ -0,0 +1,128 @@
+version: "3.8"
```

Comment:
Since so much of this is duplicated from the [default compose](https://github.com/aws/amazon-mwaa-docker-images/blob/main/images/airflow/2.9.2/docker-compose.yaml) can we move everything shared to a base compose file and [extend](https://docs.docker.com/compose/multiple-compose-files/extends/) for the split/hybrid worker options

Reply:
Sure, I'll refactor this

Reply:
So turns out docker-compose lost support for extends as of v3. The alternative is to use file overrides like `docker-compose -f "base.yaml" -f "override.yaml"`. However, this breaks the yaml anchor references, unless we redefine them across all files. There is a workaround discussed [here](https://github.com/docker/compose/issues/5621#issuecomment-581797428) by concating the files and piping to docker-compose, but I've not been able to get that to work due to duplicate definitions.  

I think there is a way to achieve what we want, but may require more research and a larger refactor of the compose structure. I would prefer to leave this out of scope for this PR.

Reply:
Yeah, I think it is fine to leave this for another PR. To be honest, personally, I don't like the limitations of Docker and Docker Compose when it comes to programmability, and this is why for the Dockerfiles, I ended up using a templating engine (Jinja). For Docker Compose, if the need arises for more sophistication, we can think of a solution later, ideally a mechanism that automatically converts Fargate service definitions into Docker Compose, or some DSL that automate the generation of both Fargate services and Docker Compose.